docs/topics_5/rn-resolved-issues.adoc

```diff
@@ -5,14 +5,26 @@
 
 At the time of the release the following resolved issues have been identified as major issues worth highlighting.
 
-For a full list of all issues resolved in this release, see the list of link:https://issues.redhat.com/issues/?filter=12356591[{ProductVersion} resolved issues] in Jira.
+For a full list of all issues resolved in this release, see the list of link:https://issues.redhat.com/browse/WINDUP-2759?filter=12357835[{ProductVersion} resolved issues] in Jira.
```

Comment:
I see a problem here. Phil named the filter "5.1.1 Resolved issues" but it is 5.1.0.Final and 5.1.1. Ask him to change this to "5.1 resolved issues" ASAP. Change the link text from `{ProductVersion} resolved issues` to `{DocInfoProductNumber} resolved issues`.